backend/src/api/comment/create_comment.rs

```diff
@@ -120,4 +166,53 @@ mod test {
             assert_eq!(text, comment_text);
         });
     }
+
+    #[test_case("LMAO !pogu", "LMAO __pog__u__url__", vec![("pogu","__pog__u__url__")])]
+    #[test_case("LMAO!", "LMAO!", vec![("pogu","__pog__u__url__")])]
+    #[test_case("LMAO !pogu !clueless XDX!!!!!", "LMAO __pog__u__url__ __clueless_url__ XDX!!!!!", vec![("pogu","__pog__u__url__"), ("clueless", "__clueless_url__")])]
```

Comment:
Maybe a test case for using a non-existing alias, and one for using the same one twice.

Reply:
Could also maybe make the lines less long by using shorter alias names and content :)